packages/eas-cli/src/commands/update/index.ts

```diff
@@ -577,6 +602,38 @@ export default class UpdatePublish extends EasCommand {
       }
     }
   }
+
+  private sanitizeFlags(flags: RawUpdateFlags): UpdateFlags {
+    const nonInteractive = flags['non-interactive'] ?? false;
```

Comment:
```suggestion
    const nonInteractive = flags['non-interactive'];
```
(since this is type boolean)

Reply:
Sure, no strong opinions on this one. I just added it for clarity; this is the default (just like we do with some other flags). But it is indeed very explicit.

Reply:
Ah, the `this.getContextAsync` requires a boolean for `nonInteractive`. Since we have to provide a default anyways, I'll revert this change and put it in the `santizeFlags` method.